minigalaxy/api.py

```diff
@@ -224,3 +225,32 @@ def __request(self, url: str = None, params: dict = None) -> dict:
             print("Response body: {}".format(response.text))
             print("")
         return response.json()
+
+    @staticmethod
```

Comment:
Why is this static? It uses the session, right? Same above here.

Reply:
It is static, because it doesn't use "self" anywhere. As a result it can be placed outside of the class with no consequences. PyCharm marks such a things as violation of PEP-8 and wants to mark method as static or place it as a function outside of the class.